app/views/requests/new.html.erb

```diff
@@ -3,8 +3,8 @@
 
 <% breadcrumb :request, @request %>
 
-<%= semantic_form_for @request, url: { action: "create" }, html: { novalidate: false } do |f| %>
+<%= form_for @request, url: { action: "create" }, html: { novalidate: false } do |f| %>
   <%= render partial: "request_details", locals: { f: f } %>
 
-  <%= f.action :submit, label: "Submit", button_html: { class: "btn btn-success" } %>
+  <%= f.submit value: "Submit", class: "btn btn-success" %>
```

Comment:
I think you can just do `f.submit, class: "whatever"` or maybe `f.submit :submit, class: "whatever`